Readme.md

```diff
@@ -90,6 +90,9 @@ rake test:all
 
 ## Changelog
 
+master
+
+* Fixed name collision of `set_locale` around action breaking Rails mailer previews.
```

Comment:
Added support for Rails 5.2

Reply:
Added mention or Rails 5.2. Testing against 5.2 was added in #12.